DaoBuilder/dcntDAOConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "ethers";
+import { DecentDAOConfig } from "./types";
+
+export const decentDAOConfig: DecentDAOConfig = {
+  lockStart: Math.floor(Date.now() / 1000), // (Now) Start time of token lock
+  lockDuration: 60 * 60 * 24 * 365, // 1 year
```

Comment:
these calculations look correct to me (though `technically` should be 365.25), though we are doing 3 years, I believe

`lockStart` is divided by 1000 because `Date.now()` is in _milliseconds_, but Ethereum timestamps are in `seconds`

no such adjustment to `lockDuration`, because that is calculating seconds